src/app/api/_api-services/redis_service/index.ts

```
@@ -618,4 +630,72 @@ export class RedisService {
+		analyticsType
+	}: {
+		network: string;
+		proposalType: string;
+		indexOrHash: string;
+		data: IPostTilesVotes;
+		votesType: EPostTilesVotesType;
+		analyticsType?: EAnalyticsType;
+	}): Promise<void> {
+		const isActivePost = data.proposal?.status && ACTIVE_PROPOSAL_STATUSES.includes(data.proposal.status as EProposalStatus);
+
+		await this.Set({
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)